src/GraphQL/Query/Operator/Thumbnail.php

```diff
@@ -64,7 +64,7 @@ public function getLabeledValue($element, ResolveInfo $resolveInfo = null)
             if ($childResult->value instanceof Asset\Image || $childResult->value instanceof Asset\Video) {
                 $childValue = $result->value = $childResult->value;
                 $thumbnail = $childValue->getThumbnail($this->thumbnailConfig, false);
-                $result->value = $thumbnail->getPath(false);
+                $result->value = $thumbnail->getPath(['deferredAllowed' => false]);
```

Comment:
Adjust the method according to the changes signature. See https://github.com/pimcore/pimcore/pull/14616